backend/src/zango/api/platform/tasks/v1/views.py

```diff
@@ -70,6 +78,7 @@ def post(self, request, app_uuid, *args, **kwargs):
             tenant = TenantModel.objects.get(uuid=app_uuid)
             connection.set_tenant(tenant)
             with connection.cursor() as c:
+                ws = Workspace.get_plugin_source()
```

Comment:
@deepakdinesh1123 Is this needed over here? because below we are creating new Workspace instance